app/_src/gateway/upgrade-31x.md

```diff
@@ -207,3 +210,38 @@ Perform a rolling upgrade of your cluster:
    is going smoothly.
 8. When your traffic is fully migrated to the 3.1.x cluster,
    decommission your old 3.0.x data planes.
+
+## Upgrade from 2.8.x.x to 3.1.1.2 
+
+### Traditional mode
```

Comment:
It occurs to me that we list DB-less as one of the modes in the table above, but don't have a migration path in here. No one's ever mentioned it though - I wonder why?